icons/send-to-back.svg

```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="14" y="14" width="8" height="8" rx="2" />
```

Comment:
I liked it better before.

Reply:
> I liked it better before.

@jguddas 😅 Valid, but it isn’t consistent, or accurate… Maybe we ultimately decide that doesn’t matter? cc: @karsa-mistmere @ericfennis

Reply:
I'm also with @jguddas other looked better. To me 100% it looks like two squares with connecting lines. The corners are tough to see in this adjustment.

Reply:
I also agree, my train of thought was the same as that of @ericfennis, while it's less consistent with its pair, the original design is a bit more legible.

Reply:
> The corners are tough to see in this adjustment.

Another option would be to make them slightly sharper to compensate (as in third below)… Here are all the possible options for comparison:

![preview](https://github.com/lucide-icons/lucide/assets/7797479/450abe73-254b-425d-8d10-171cde3d4a5e)

@jguddas @ericfennis @karsa-mistmere Thoughts?

Reply:
My vote is still for the original.

Reply:
I vote for 1 and 3.

Reply:
I think 3. So looks like @karsa-mistmere has the deciding vote…

Reply:
I'm also voting for the original — back when that icon was submitted I'd ruminated over posting a suggestion for no. 3 myself, so I definitely see where you're coming from, but I think all things considered, no. 1 simply has much better legibility.

We could check if alternatively, making `bring-to-front` more consistent with this one would work better (i.e. increasing the central square in that one).

Reply:
Column 1: current icons
Column 2: `bring-to-front` w/ larger rectangle to match `send-to-back`

![image](https://github.com/lucide-icons/lucide/assets/17746067/522576e6-a002-4260-a00b-1c651eeec370)
![image](https://github.com/lucide-icons/lucide/assets/17746067/f0a2d2bd-ccb4-4f53-9aae-e6da400e27c6)

Personally, I'm fine with either.

Reply:
I'm fine with making `bring-to-front` bigger.

Reply:
> We could check if alternatively, making `bring-to-front` more consistent with this one (i.e. increasing the central square in that one) would work better.

I think that makes that one look much worse 😅 `reverted`, so just the metadata improvements then…